src/devices/LidarLiteV3/LidarLiteV3.cs

```diff
@@ -0,0 +1,337 @@
+            {
+                throw new ArgumentOutOfRangeException("Count must be between 2 and 254");
+            };
+
+            switch (measurementRepetition)
+            {
+                case MeasurementRepetition.Repeat:
+                    WriteRegister(Register.OUTER_LOOP_COUNT, (byte)count);
+                    break;
+                case MeasurementRepetition.RepeatIndefinitely:
+                    WriteRegister(Register.OUTER_LOOP_COUNT, 0xff);
+                    break;
+                case MeasurementRepetition.Off:
+                    WriteRegister(Register.OUTER_LOOP_COUNT, 0x00);
+                    break;
+            }
+
+            if (delay != null)
+            {
+                WriteRegister(Register.MEASURE_DELAY, (byte)delay.Value);
+
+                // Set mode to use custom delay.
+                var currentAcqSettings = AcquistionSettings;
+                currentAcqSettings |= AcquistionSettings.UseDefaultDelay;
+                AcquistionSettings = currentAcqSettings;
+            } else
+            {
+                // Set mode to use default delay.
+                var currentAcqSettings = AcquistionSettings;
+                currentAcqSettings &= ~AcquistionSettings.UseDefaultDelay;
+                AcquistionSettings = currentAcqSettings;
+            }
+
+            // Kick it off with a single acquire command.
+            WriteRegister(Register.ACQ_COMMAND, 0x04);
+        }
+
+        /// <summary>
+        /// Set a new I2C address.
+        /// </summary>
+        /// <param name="address">new address, valid values are 7-bit values with 0 in the LSB.</param>
+        public void SetI2CAddress(byte address)
+        {
+            if ((address & 1) == 1)
+            {
+                throw new ArgumentOutOfRangeException("Address must have 0-bit in the LSB.");
+            }
+
+            // Read in the unit's serial number.
+            Span<byte> rawData = stackalloc byte[2] { 0, 0 };
+            ReadBytes(Register.UNIT_ID, rawData);
+
+            // Write serial number to I2C_ID.
+            WriteRegister(Register.I2C_ID_HIGH, rawData[1]);
+            WriteRegister(Register.I2C_ID_LOW, rawData[0]);
+
+            // Write the new address.
+            WriteRegister(Register.I2C_SEC_ADDR, address);
+
+            // Disable the default address
+            WriteRegister(Register.I2C_CONFIG, 0x08);
+        }
+
+        #region Device Registers
+
+        /// <summary>
+        /// Get the last distance measurement in cm.
+        /// </summary>
+        public int LastDistance 
+        {
+            get {
+                Span<byte> rawData = stackalloc byte[2] { 0, 0 };
+                ReadBytes(Register.FULL_DELAY, rawData);
+                return BinaryPrimitives.ReadUInt16BigEndian(rawData);
+            }
+        }
+
+        /// <summary>
+        /// Get the difference between the current and last measurement resulting
```

Comment:
There is something off about this description. Velocity should be in time domain so:
- should time between measurements be accounted for here?
- should the unit be cm/s cm/h?

If this is just pure difference then it should not be called Velocity...

Reply:
This one is tricky, it's the difference calculated between the last two distance reading, which you're right, isn't velocity.  

However, when used in conjection with MeasurementRepeititon.RepeatIndefinitely and a delay, it can be used to reliably calculate a velocity from the device.  The default delay equals velocity measurements in 0.1 m/s.  Since the delay is configurable, the unit isn't defined.  I believe this is the intented use for this register (which is why it's called velocity).

While you can do this calculation outside of the device in C#, I persume it would be more precise to do it on the device itself.

I'm going to rename the property to `DifferenceBetweenLastTwoDistances`.